cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/common/tools/ExceptionHandler.java

```diff
@@ -103,28 +103,23 @@ private static void openInvalidConfigurationErrorDialog(InvalidConfigurationExce
   /**
    * Opens up a message dialog for displaying further guidance on upgrading old templates.
    *
-   * @param e {@link InvalidConfigurationException} occurred
+   * @param e {@link DeprecatedMonolithicConfigurationException} occurred
    */
-  private static void openOldTemplatesErrorDialog(DeprecatedMonolithicTemplatesException e) {
+  private static void openMonolithicConfigurationErrorDialog(DeprecatedMonolithicConfigurationException e) {
 
     PlatformUIUtil.getWorkbench().getDisplay().syncExec(() -> {
       MessageDialog dialog = new MessageDialog(Display.getDefault().getActiveShell(), "Warning!", null,
-          "You are using an old templates version. Do you want to upgrade your templates to the newest version? "
-              + "For more informations please visit:" + WikiConstants.WIKI_UPGRADE_OLD_TEMPLATES
-              + "\n\nOriginal error message: " + e.getMessage(),
-          MessageDialog.ERROR, new String[] { "Upgrade", "Postpone", "Cancel" }, 2);
```

Comment:
Don't we need a Cancel button here too?

Reply:
No, I think it does not matter that much, because before generating there is a cancel button there.